tests/functional/simple_seed/test_seed.py

```diff
@@ -1,19 +1,290 @@
+
+
+class TestSeedConfigFullRefreshOn(SeedTestBase):
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {"quote_columns": False, "full_refresh": True},
+        }
+
+    def test_postgres_simple_seed_full_refresh_config(self, project):
+        """config option should drop current model and cascade drop to downstream models"""
+        self._build_relations_for_test(project)
+        self._check_relation_end_state(run_result=run_dbt(["seed"]), project=project, exists=False)
+
+
+class TestSeedConfigFullRefreshOff(SeedTestBase):
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {"quote_columns": False, "full_refresh": False},
+        }
+
+    def test_postgres_simple_seed_full_refresh_config(self, project):
+        """Config options should override full-refresh flag because config is higher priority"""
+        self._build_relations_for_test(project)
+        self._check_relation_end_state(run_result=run_dbt(["seed"]), project=project, exists=True)
+        self._check_relation_end_state(
+            run_result=run_dbt(["seed", "--full-refresh"]), project=project, exists=True
+        )
+
+
+class TestSeedCustomSchema(SeedTestBase):
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
+        """Create table for ensuring seeds and models used in tests build correctly"""
+        project.run_sql_file(project.test_data_dir / Path("seed_expected.sql"))
+
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {
+                "schema": "custom_schema",
+                "quote_columns": False,
+            },
+        }
+
+    def test_postgres_simple_seed_with_schema(self, project):
+        results = run_dbt(["seed"])
+        assert len(results) == 1
+        # TODO: 1/4 custom schema work; but unsure how to provide these lines with custom schema on seed_actual
+        # check_relations_equal(project.adapter, ["seed_actual", "seed_expected"])
+
+        # this should truncate the seed_actual table, then re-insert
+        results = run_dbt(["seed"])
+        assert len(results) == 1
+        # TODO: 2/4
+        # check_relations_equal(project.adapter, ["seed_actual", "seed_expected"])
+
+    def test_postgres_simple_seed_with_drop_and_schema(self):
+        results = run_dbt(["seed"])
+        assert len(results) == 1
+        # TODO: 3/4
+        # check_relations_equal(project.adapter, ["seed_actual", "seed_expected"])
+
+        # this should drop the seed table, then re-create
+        results = run_dbt(["seed", "--full-refresh"])
+        results = run_dbt(["seed"])
+        # TODO: 4/4
+        # check_relations_equal(project.adapter, ["seed_actual", "seed_expected"])
+
+
+class TestSimpleSeedEnabledViaConfig(object):
```

Comment:
@gshank @VersusFacit How do you guys feel about moving this to selection(graph selection?) tests? since this is mainly testing the selection functionality

Reply:
I could weigh in better if I understood the primacy of the selection feature. Do we consider it a subfeature of seed or do we consider seed a command which mixes in the selection behavior. I feel like that reasoning best informs whether it should be moved to a selection test or not.